asmcomp/mach.ml

```diff
@@ -63,6 +67,7 @@ type operation =
   | Ispecific of Arch.specific_operation
   | Iname_for_debugger of { ident : Backend_var.t; which_parameter : int option;
       provenance : unit option; is_assignment : bool; }
+  | Ipollcall of { check_young_limit: bool }
```

Comment:
What is the purpose of `check_young_limit`? Is `Ipollcall { check_young_limit = false }` correspond to no polling?